gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/Project.java

```diff
@@ -40,12 +40,7 @@
 
   public static class Builder extends ProjectInfo.Builder {
     private final ResourceManager resourceManager;
-    private ProjectInfo.BuilderImpl infoBuilder;
-
-    Builder(ResourceManager resourceManager) {
-      this.resourceManager = resourceManager;
-      this.infoBuilder = new ProjectInfo.BuilderImpl();
-    }
+    private final ProjectInfo.BuilderImpl infoBuilder;
```

Comment:
can you add a comment in the issue description suggesting that resource manager is also changed in this PR?